Formula/jenkins-lts.rb

```diff
@@ -6,13 +6,19 @@ class JenkinsLts < Formula
 
   bottle :unneeded
 
-  depends_on :java => "1.8"
+  depends_on "openjdk@11"
 
   def install
-    system "jar", "xvf", "jenkins.war"
+    system Formula["openjdk@11"].opt_bin "/jar", "xvf", "jenkins.war"
```

Comment:
```suggestion
    system Formula["openjdk@11"].opt_bin/"jar", "xvf", "jenkins.war"
```

Reply:
`brew style` hated that

Reply:
Well it's syntactically incorrect now.

Reply:
Bit weird that `brew style` would fail with my suggestion though since this is how we do it for many Python invokations.

Reply:
I'll make it a full string